### PR TITLE
protean blob chameleon clothing

### PR DESCRIPTION
**Pull request by lolman360**
fetish code

### CODE REVIEW

File: code/modules/hardsuits/suits/species/protean.dm
Comment (TheLordME) [inline code/modules/hardsuits/suits/species/protean.dm:1]: very important change

**Review by TheLordME — CHANGES_REQUESTED** [review]

**Review by TheLordME — APPROVED** [review]

File: code/modules/mob/holder.dm [hunk 1]
```diff
@@ -160,6 +155,15 @@
...
 
+/obj/item/holder/protoblob
+	clothing_flags = ALLOWINTERNALS
+	slot_flags = SLOT_HEAD | SLOT_OCLOTHING | SLOT_HOLSTER | SLOT_ICLOTHING | SLOT_ID | SLOT_MASK | SLOT_GLOVES | SLOT_BACK | SLOT_BELT | SLOT_FEET | SLOT_EARS | SLOT_EYES
+	w_class = ITEMSIZE_TINY
+	allowed = list(/obj/item/gun,/obj/item/flashlight,/obj/item/tank,/obj/item/suit_cooling_unit,/obj/item/melee/baton)
+
+
+
+
```
Comment (timothyteakettle) [inline code/modules/mob/holder.dm:166]: is there a reason we have 4 newlines here

File: code/modules/species/protean/protean_blob.dm [hunk 2]
```diff
@@ -581,6 +592,25 @@
...
+			if(BP_L_FOOT)
+				targeted_area = SLOT_ID_SHOES
+				target_displayname = "feet"
+			if(BP_R_FOOT)
+				targeted_area = SLOT_ID_SHOES
+				target_displayname = "feet"
+			if(O_EYES)
+				targeted_area = SLOT_ID_GLASSES
+				target_displayname = "eyes"
+
```
Comment (timothyteakettle) [inline code/modules/species/protean/protean_blob.dm:613]: you dont need 2 newlines here

**Comment by timothyteakettle** [comment]
other than an excessive use of newlines this is fine

**Review by timothyteakettle — APPROVED** [review]

**Pull request merged**